modules/rest-api/pages/rest-bucket-create.adoc

```diff
@@ -196,19 +196,19 @@ No object is returned.
 [#ramQuota]
 === ramQuota
 
-The amount of memory to be allocated to the bucket, per node, in megabytes.
-The minimum amount is 100 MB.
+The amount of memory to be allocated to the bucket, per node, in MiB.
+The minimum amount is 100 MiB.
 The maximum amount is the total Data Service memory quota configured per node, minus the amount already assigned to other buckets.
 For information on per node memory configuration, see the page for xref:manage:manage-settings/general-settings.adoc[General] Settings.
 
 A value for `ramQuota` _must_ be specified: the value _can_ be modified, following bucket-creation.
 
-An incorrect memory-specification returns a notification such as `{"ramQuota":"RAM quota cannot be less than 100 MB"}`.
+An incorrect memory-specification returns a notification such as `{ramQuota":"RAM quota cannot be less than 100 MiB"}`.
```

Comment:
The " symbol shouldn't have been removed here

Reply:
Thank you. I have fixed.